pkg/loki/loki.go

```diff
@@ -743,7 +745,8 @@ func (t *Loki) setupModuleManager() error {
 		Distributor:              {Ring, Server, Overrides, TenantConfigs, PatternRingClient, PatternIngesterTee, Analytics, PartitionRing},
 		IngestLimitsRing:         {RuntimeConfig, Server, MemberlistKV},
 		IngestLimits:             {MemberlistKV, Server},
-		IngestLimitsFrontend:     {IngestLimitsRing, Overrides, Server},
+		IngestLimitsFrontend:     {IngestLimitsFrontendRing, IngestLimitsRing, Overrides, Server},
```

Comment:
The Frontend does not need to init the IngestLimitsFrontendRing but rather the MemberlistKV. IngestLimitsFrontendRing should be initialized by the distributor.